src/website/app/pages/Color/ColorRampRow.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import colorable from 'colorable';
+import readableColor from 'polished/lib/color/readableColor';
+import { createStyledComponent, color } from '../../../../utils';
+
+type Props = {
+  className?: string,
+  name: string
+};
+
+const styles = {
+  row: ({ theme, color }) => ({
+    backgroundColor: color,
+    display: 'flex',
+    justifyContent: 'space-between',
+    listStyle: 'none',
+    minWidth: 200,
+    padding: theme.spacing_single,
+    fontSize: theme.fontSize_ui,
+
+    '& > div > p:first-child': {
+      margin: `0 0 ${theme.spacing_single} 0`
+    },
+    '& > div > p:last-child': {
+      margin: 0
+    }
+  }),
+  colorName: ({ color }) => ({
+    color,
+    fontWeight: 'bold'
+  }),
+  hsl: ({ color }) => ({
+    color,
+    fontWeight: 100
+  }),
+  whiteOnBkd: { color: 'white' },
```

Comment:
What does `Bkd` stand for?  Background? Baked? Backend?  Please try to use easily comprehensible abbreviations.  Consider `Bg, Bkgd, Bkgnd`, etc.  If there is no such thing, just use the whole word or try removing the vowels.  Clarity is more important than concision.